app/serializers/yelp_serializer.rb

```diff
@@ -0,0 +1,21 @@
+class YelpSerializer < ActiveModel::Serializer
+  attributes :name, :address, :business_type, :thumbnail, :yelp_link
+
+  def self.search_results(results)
+    {
+      data:
+        results.map do |result|
```

Comment:
sweet! 🍭

Reply:
I think if you wanted to, you could also use fast_json here and not have to handroll the serialization. I think the ActiveModel::Serializer was actually not the best option for this project, because we're not ever serializing models really (though Jermaine will be doing that for the onboard page)

Reply:
decide as a group? i didn't really know how the active model serializer worked, but maybe atm it is good enough for mvp?